src/app/core/user/user.model.spec.js

```diff
@@ -254,6 +254,7 @@ describe('User Model:', () => {
 					providerData: {
 						dn
 					},
+					teams: undefined,
```

Comment:
Since the UserModel was updated to track the teams, and the auditCopy function was updated to include the team information, it may be worth adding a unit test here to ensure it is properly populating teams in the audit copy in the same way it did for TeamUserModel.

For example, it looks like the original TeamUser audit copy was doing a join with the Team collection to populate the team names for the user: https://github.com/bluehalo/node-rest-starter/pull/227/files#diff-e20a9b83a2a0b58a95f0f23a0cb430b976788565539703001f010f9343251ce8L167-L198